### ISSUE TITLE
Error opening .lua file (Support for WINDOWS-1252 encoding)

### ISSUE THREAD

**Issue by sonaldo** [thebaselab/codeapp#175]
when I try to open a .lua file I get the following error "unsupported file". I think it may be due to the encode, the file is in "WINDOWS-1252".

**Comment by bummoblizard** [comment]
Closing as I've added support for WINDOWS-1252 encoding in the latest App Store release. Feel free to reopen if the issue persists.

**Comment by sonaldo** [comment]
Hi, I still can't open the file here, I used textastic and it opens normally there.

**Comment by bummoblizard** [comment]
I see. It would be great if you can send me an example file to demonstrate
the problem.

**Comment by sonaldo** [comment]
Yes i can, https://www.icloud.com/iclouddrive/0AgSBJaC_nITtIkRO7e0n0TVQ#config

Thanks.
